### ISSUE TITLE
Need a new name for "Re-Roll"

### ISSUE THREAD

**Issue by SabreCat** [HabitRPG/habitica#1938]
With Classes coming up, there are going to be several features that conceptually map to the tabletop RPG or MMO concept of "re-rolling": account reset, class change, and the actual Re-Roll feature currently available in the Reward shop. That could lead to some user confusion.

Moreover, the semantics aren't a great match to begin with: "re-roll" has its origins in "rolling up a character," referring to rolls of dice to randomly determine an RPG character's beginning stats. There's nothing random about the current Re-Roll, which is more about returning a user's Dailies and Habits to a baseline danger level.

What, then, would make a better name for this? Braindump...
- Cool Off (possibly too close a cognate to MMO "cooldown")
- Refresh (hrm, no, that sounds like the browser action)
- Breather / Catch Your Breath
- Reset (even if we change the name of the current "Reset" feature, this sounds like it'd wipe everything out)
- Panic or Panic Button (heh)
- Full Defense
- Guard

**Comment by lefnire** [comment]
I think @wc8 had a suggestion like "back to yellow". We could also put it in Options > Settings along with restore et al. It doesn't make much sense as a store item i suppose... (random FYI: not even meant to be a monetization strategy, I only charge gems so people don't abuse it). Also, I think it may be less necessary once we introduce spells for reducing redness, as long as people keep up with their class.

All this is to say, let's tuck it away and - as you said - rename it.

**Comment by SabreCat** [comment]
I think it's not so bad as a minor pay-for feature. It's basically a convenience option, because you can do the exact same thing by deleting and recreating those tasks. It just saves you a hunk of time for doing so. But aye, moving it to another tab would make sense!

**Comment by ShadeJackrabbit** [comment]
I like the line of "Guard", though that sounds too close to a targetted action. What about "Fortify"?

**Comment by SabreCat** [comment]
@ShadeJackrabbit: I like it!

**Comment by wc8** [comment]
A clear explanation will still be needed with _Fortify_. 
Fortify sounds more appealing than re-roll, but still doesn’t describe what will happen: all tasks turning back to yellow. 
(I think what I had suggested (off the top of my head) was “Clean Slate”.)

**Comment by SabreCat** [comment]
@wc8: Agreed. Can you take a look at the changed files in [the fix commit](https://github.com/HabitRPG/habitrpg/commit/af80548a342fdaa636de24a5e0aee7a32b1508db)? You should be able to see what's a tooltip popup, what's a description in a modal window, etc.--let me know what questions and suggestions you come up with.

**Comment by francehopper** [comment]
Both "Guard" and "Fortify" sound good to me. Have you considered "Defend" though? Given that Re-roll kinda-sorta reset stats, I'm not sure that would work the best though.

**Comment by SabreCat** [comment]
"Defend" has the same problem as "Guard," in that it's most commonly used as a transitive verb in plain English--you guard _something_, defend _someone_, which doesn't click here. Classic jRPG gamers would see the connection pretty quick (oh, yeah, it's that thing I do when I skip my turn to take less damage!), but more and more we're finding our user base isn't actually fluent in those tropes.

**Comment by francehopper** [comment]
Ah. Good point there. This is more for sarcasm than to be taken as an actual suggestion, but how about "Hunker Down"? (meant in this context at about 1:30: http://www.youtube.com/watch?v=PAXxPb4uNOQ) As an actual suggestion, could "Rest" potentially work?

**Comment by SabreCat** [comment]
Heh! Hunker Down is an action in the recent XCOM games, and actually not so far off from the concept... but as conceived in the current pull request for this, it's a service sold in the shop, so having Alexander "hunker me down" is a bit weird. "Fortify" is still our front-runner.

"Rest," unfortunately no, we've already got one of those. "Rest in the Inn" puts you into vacation mode where Dailies don't damage you at all.

I just noticed I didn't respond to @wc8's "Clean Slate" suggestion directly--some of my thoughts ran that direction too, but it sounds too much like an Account Reset type of action, like it'd delete all your tasks or start you over.

**Comment by wc8** [comment]
@SabreCat +1 for Fortify
I’ll have to look up the definition of jRPG, but I think it’s a plus that Fortify has roots in an RPG game different from commonly mentioned RPGs. A salute here and there to other RPGs is a nice touch.

I checked the pull request, I think the mouseover text on the button is clearer (hopefully it will be easier to access).

I know this is a little out of the scope of this ticket, but would it be possible to add a notification _post_ Fortify? 
That is, the user gets a **notification** message **after** spending the gems, confirming the gem purchase took effect.
e.g. _You will take less damage when skipping your turn_ or _You crossed paths with a benevolent elf and your dailies will no longer deal as much damage._ 
From [Click Feedback & Confirmation](https://trello.com/c/uEo7wjkH)
[Updated suggested text to remove re-roll references, but it’s still just placeholders for what the notification might say.]

**Comment by SabreCat** [comment]
@wc8: jRPG = Japanese (video game) RPG = Final Fantasy, Dragon Warrior, and many others, which tend to be the main inspiration for pixely Habit. I was referring to "Defend", there; I can't immediately think of a game "Fortify" was used in (maybe Puzzle Quest?).

Re post-click notification, that's a possibility, though it'll take me a little bit of work to figure out how, newb that I am.

**Comment by SabreCat** [comment]
There's not a post-click modal, but it does take you back to the main screen, and you can see all your tasks change color when the page updates. I think this is done; closing.
